desk/app/channels.hoon

```diff
@@ -1446,10 +1537,38 @@
       =/  post  (get:on posts.channel `@da`time)
       ?~  post  ~
       ?~  u.post  `~
-      (ca-peek-replies id.u.u.post replies.u.u.post rest.pole)
+      (ca-peek-replies-1 id.u.u.post replies.u.u.post rest.pole)
+    ==
+  ::
+  ++  ca-peek-replies-0
+    |=  [parent-id=id-post:c replies=v-replies:c =(pole knot)]
```

Comment:
I think these are giving the new type not the old

Reply:
It looks like they always looked like this. Maybe that was wrong, though?

Reply:
right but we changed what those types are, the old types are now the `simple-` which we aren't giving here. just need to do something similar to what we did for `ca-peek-posts-0`

Reply:
But v-reply has always had the rev on it, and these were using v-reply (except for the individual reply scry that I switched over). Maybe we just didn't notice because we don't actually use these anywhere.

If anyone was using these, they were getting the `rev`, in other words.

Reply:
Yep

Reply:
Yeah, it fails on v1 and v0 because of the mark. Will fix.

Reply:
This should be fixed in the latest commit.